scripts/run-django-dev.sh

```diff
@@ -4,6 +4,7 @@
 
 python3 manage.py collectstatic --noinput --clear
 python3 manage.py migrate --noinput
+python3 manage.py createcachetable
```

Comment:
MIT Learn runs this as part of `run-django-dev.sh`; it exits successfully even if already set up.